lib/rules/cli/4.5.yaml

```diff
@@ -1091,6 +1091,7 @@
   :options:
     :buildconfig: <value>
     :build_arg: --build-arg=<value>
+    :build_loglevel: --build-loglevel=<value>
```

Comment:
How about 4.4.yaml file, do we need update this too?

Reply:
Yes, missed it, Updated 4.4 yaml now.